### PR TITLE
Add {Int,Long,Float,Double,Bool}#toStringBinary

### PR DESCRIPTION
**Pull request by soc**
This change adds a helper function that's helpful when working with bits.

This is certainly not the most elegant, elaborate or flexible approach
(compared to languages that support arbitrary bases or compared to a
typeclass-based design that enables a maximum of abstraction), but it is
simple and straight-forward and should last until a better solution is
required.

### CODE REVIEW

File: dora/stdlib/Double.dora [hunk 1]
```diff
@@ -4,6 +4,16 @@
   @internal fun toFloat() -> Float;
   @internal fun toString() -> String;
 
+  fun toStringBinary() -> String {
```
Comment (dinfuehr) [inline dora/stdlib/Double.dora]: Couldn't we share all those implementations in a single function (apart from bool)? Might also be a good use-case for the StringBuffer class.

**Comment by dinfuehr** [comment]
Agreed, hex-strings would be nice as well.

**Comment by soc** [comment]
> Agreed, hex-strings would be nice as well.

Yes, had that in mind too!

File: dora/stdlib/Double.dora (hunk 1 above)
Comment (soc) [reply dora/stdlib/Double.dora]: Had considered it, but I didn't want to have a "public" method somewhere that was only supposed to be used internally. Where would you place it?

Comment (dinfuehr) [reply dora/stdlib/Double.dora]: I suppose in any of these files is good enough for now. That this function is effectively `public` for now is definitely unfortunate, let's hope we will have some form of encapsulation sooner or later.

Comment (soc) [reply dora/stdlib/Double.dora]: Alright, I updated the PR.

**Comment by dinfuehr** [comment]
Cool, thanks!

**Pull request merged**